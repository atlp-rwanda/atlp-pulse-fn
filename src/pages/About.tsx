/* eslint-disable react/no-unescaped-entities */
import { useTranslation } from 'react-i18next';
import React, { useState, useEffect } from 'react';
import { AiOutlineLeft, AiOutlineRight } from 'react-icons/ai';
import Frame from '../assets/Frame.svg';
import second from '../assets/second.svg';
import third from '../assets/third.svg';
import fourth from '../assets/fourth.svg';
import person from '../assets/person.png';
import person2 from '../assets/person2.png';
import ur from '../assets/ur.png';

const testimonials = [
  {
    id: 1,
    name: 'Bernard Dushimimana',
    role: 'Sr.Manager',
    image: person,
    content: `I'm extremely impressed with Pulse and their performance management platform.
        Since using their services, it has been a game-changer for our organization.
        The platform is intuitive, easy to navigate, and packed with powerful features.
        Tracking performance metrics and setting goals has never been easier. Real-time
        data and analytics provided by Pulse offer valuable insights into our team's
        performance, enabling data-driven decisions. The customer support team has been
        highly responsive and knowledgeable, addressing our queries promptly. Pulse has
        transformed our performance management process, optimizing our operations and
        driving meaningful results. I highly recommend Pulse to any business seeking to
        elevate their performance management to the next level.`,
  },
  {
    id: 2,
    name: 'Susan',
    role: 'Sr.Manager',
    image: person2,
    content: `I'm delighted to share my positive experience with Pulse and their exceptional
        performance management platform. Implementing their services has led to remarkable
        improvements in our performance tracking and management processes. The user-friendly
        interface and comprehensive features have simplified our goal setting and progress
        monitoring activities. Pulse's in-depth analytics have provided valuable insights
        into our team's performance trends, enabling informed decisions and continuous growth.
        Thanks to Pulse, we have achieved greater transparency, accountability, and productivity
        within our organization. I highly recommend Pulse to any company seeking a robust and
        reliable performance management solution.`,
  },
  {
    id: 3,
    name: 'University of Rwanda',
    role: 'organisation',
    image: ur,
    content: `
        We are thrilled with the services provided by Pulse. Their performance management platform
        has exceeded our expectations in every way. The user-friendly interface and comprehensive
        features have made tracking and monitoring our performance metrics a breeze. Pulse's
        data-driven insights have revolutionized our approach to performance management,
        enabling informed decisions and continuous improvement. The support team at Pulse has
        been exceptional, offering prompt and knowledgeable assistance whenever needed.
        I wholeheartedly endorse Pulse to any organization seeking to optimize their performance
        management process and achieve outstanding results.
        `,
  },
];

function Testimonial() {
  const [currentIndex, setCurrentIndex] = useState(0);

  const prevTestimonial = () => {
    setCurrentIndex(
      (prevIndex) =>
        (prevIndex - 1 + testimonials.length) % testimonials.length,
    );
  };

  const nextTestimonial = () => {
    setCurrentIndex((prevIndex) => (prevIndex + 1) % testimonials.length);
  };

  useEffect(() => {
    const interval = setInterval(() => {
      setCurrentIndex((prevIndex) => (prevIndex + 1) % testimonials.length);
    }, 10000);

    return () => clearInterval(interval);
  }, []);

  return (
    <>
      <div className="block md:hidden flex-col lg:mx-10 md:mx-20 my-20 gap-10 relative ">
        {testimonials.map((testimonial, index) => (
          <div
            key={testimonial.id}
            className={`bg-indigo-100 dark:bg-dark-bg  dark:text-slate-300 lg:w-1/3 p-8 md:w-full rounded-b-3xl sm:mx-3 rounded-t-3xl ${
              currentIndex === index ? 'visible' : 'hidden'
            }`}
          >
            <div className="flex flex-col sm:flex-row  mb-4 items-center">
              <img
                className="md:w-1/4 sm:w-1/3"
                src={testimonial.image}
                alt=""
              />

              <ul>
                <li className="text-xs mt-4 ml-3  dark:text-slate-300 text-neutral-600">
                  {testimonial.name}
                </li>
                <li className="text-xs mt-2  dark:text-slate-300 ml-3">
                  {testimonial.role}
                </li>
              </ul>
            </div>
            <p className="card-text  dark:text-slate-300 text-neutral-900">
              {testimonial.content}
            </p>
          </div>
        ))}

        <span className="absolute top-1/2 left-3 transform -translate-y-1/2  text-neutral-600 rounded-full bg-slate-50 p-1 drop-shadow-2xl opacity-50 ">
          <AiOutlineLeft
            size={20}
            onClick={prevTestimonial}
            className="cursor-pointer"
          />
        </span>

        <span className="absolute top-1/2 right-3 transform -translate-y-1/2 text-neutral-600 rounded-full bg-slate-50 p-1 drop-shadow-2xl opacity-50 ">
          <AiOutlineRight
            size={20}
            onClick={nextTestimonial}
            className="cursor-pointer"
          />
        </span>
      </div>

      <div className="hidden md:flex md:flex-row flex-col  lg:mx-0 md:mx-0 my-20 gap-10 md:flex-wrap lg:flex-nowrap ">
        {testimonials.map((testimonial, index) => (
          <div
            key={testimonial.id}
            className="bg-indigo-100  dark:bg-dark-bg lg:w-1/3 p-8 md:w-full  rounded-b-3xl sm:mx-3 rounded-t-3xl "
          >
            <div className="flex flex-col sm:flex-row  mb-4 items-center">
              <img
                className="md:w-1/4 sm:w-1/3"
                src={testimonial.image}
                alt=""
              />

              <ul>
                <li className="text-xs mt-4 ml-3  dark:text-slate-300  text-neutral-600">
                  {testimonial.name}
                </li>
                <li className="text-xs  dark:text-slate-300  mt-2 ml-3">
                  {testimonial.role}
                </li>
              </ul>
            </div>
            <p className="card-text  dark:text-slate-300  text-neutral-900">
              {testimonial.content}
            </p>
          </div>
        ))}
      </div>
    </>
  );
}

function About({ styles }: any) {
  const { t } = useTranslation();

  const aboutCards = [
    {
      title: t('performance'),
      body: t('body1'),
      img: Frame,
      orientation: 'flex-row',
    },
    {
      title: t('analytics'),
      body: t('body2'),
      img: second,
      orientation: 'flex-row-reverse',
    },
    {
      title: t('continuos'),
      body: t('body3'),
      img: third,
      orientation: 'flex-row',
    },
    {
      title: t('goal'),
      body: t('body4'),
      img: fourth,
      orientation: 'flex-row-reverse',
    },
  ];

  return (
    <div className=" bg-white   mt-auto dark:bg-dark-frame-bg">
      <div className=" justify-between w-full h-full pb-5">
        <div>
          <h1 className="text-center text-neutral-700 mb-5 font lg:text-4xl md:text-2xl pt-10 pb-4  dark:text-slate-100">
            {t('hero2')}
          </h1>
        </div>

        {aboutCards.map(({ title, body, img, orientation }) => (
          <div
            key={body}
            className={`mt-10 card-image lg:mx-10 sm:mx-5 md:flex ${orientation} `}
          >
            <div className="shadow-md sm:shadow-none mx-auto md:mx-0 bg-indigo-100  items-center  md:items-start flex-col dark:bg-dark-bg md:w-[45%] md:py-10 md:px-4 max-w-[35em] sm:w-full   sm:rounded-3xl  md:mr-2">
              <h2 className="pt-3 header-style lg:px-5 lg:text-2xl sm:text-xl lg:text-start sm:text-center md:text-start dark:text-slate-200">
                {title}
              </h2>
              <div>
                <div className="xl:hidden md:hidden mb-0 mt-4">
                  <img src={img} alt="frame" />
                </div>
                <p className="text-style mt-0 lg:px-5 bg-[#E0E7FF] dark:bg-dark-bg rounded-b-3xl md:rounded-none p-8 md:px-0 md:py-4 dark:text-slate-300">
                  {body}
                </p>
              </div>
            </div>
            <div className="sm:hidden md:flex xl:flex lg:flex">
              <img
                className="rounded-2xl dark:opacity-80 lg:max-h-[25em]"
                src={img}
                alt="frame"
              />
            </div>
          </div>
        ))}

        <div className="px-5">
          <h1 className="pt-3 text-center font text-neutral-700 lg:text-4xl sm:text-2xl  italic dark:text-slate-100">
            Excellent human resources and companies
          </h1>
          <h1 className="font text-center text-neutral-700 lg:text-4xl sm:text-2xl dark:text-slate-100">
            {' '}
            Come shape the future together{' '}
          </h1>
        </div>
        <Testimonial />
      </div>
    </div>
  );
}
export { About}
