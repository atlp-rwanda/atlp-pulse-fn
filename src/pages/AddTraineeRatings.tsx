import useDocumentTitle from '../hook/useDocumentTitle';

function AddTraineeRatings() {
  useDocumentTitle('Add Trainee Ratings');

  return null;
}

export default AddTraineeRatings;
