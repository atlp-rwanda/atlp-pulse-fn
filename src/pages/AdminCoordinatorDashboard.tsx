/* eslint-disable */
/* istanbul ignore file */

import React, { useState, useEffect,Suspense, useContext } from 'react';
import { useTranslation } from 'react-i18next';
import Paper, { PaperProps } from '@mui/material/Paper';
import Draggable from 'react-draggable';
import Skeleton from '../components/Skeleton';
import DataTable from '../components/DataTable';
import Sidebar from '../components/Sidebar';
import useDocumentTitle from '../hook/useDocumentTitle';
import { toast } from 'react-toastify';
import{ GetAllCoordinatorUsers} from '../Mutations/Coordinator';
import { useLazyQuery, } from '@apollo/client';
const organizationToken = localStorage.getItem('orgToken');

const AdminTraineeDashboard = () => {
  useDocumentTitle('Coordinators');
  const { t }: any = useTranslation();
  const [coordinators, setCoordinators] = useState<any>([]);
  const [nav, setNav] = useState(false);
  const handleClick = () => setNav(!nav);

 const data = coordinators;
  const columns = [
    { Header: t('name'), accessor: 'profile[name]' },
    { Header: t('email'), accessor: 'email' },
    { Header: t('cohort'), accessor: 'cohort[name]' },
    { Header: t('program'), accessor: 'cohort[program[name]]' },
    { Header: t('manager'), accessor: 'cohort[program[manager[profile[name]]]]' },
  ];
  const [getCoordinatorsQuery] = useLazyQuery(GetAllCoordinatorUsers, {
    variables: {
      orgToken: organizationToken,
    },
  });

  /* istanbul ignore if */

  useEffect(() => {
  
    getCoordinatorsQuery({
      fetchPolicy: 'network-only',
      onCompleted: (data) => {
        setCoordinators(data.getAllCoordinatorUsers);
        
       
      },

      onError: (error) => {
        toast.error(error.message);
      },
    });
  }, []);
  /* istanbul ignore if */

  return (
    <>



      <div className="flex flex-col h-screen">
        <div className="flex flex-row">
          <Sidebar toggle={handleClick} style="hidden lg:flex" />
          <div className="w-full">
            <div>
              <div className="bg-light-bg dark:bg-dark-frame-bg  min-h-screen overflow-y-auto overflow-x-hidden">
                <div className="flex items-left px-10 lg:px-60 pt-24 pb-8">
                  <div className="space-x-8 lg:ml-7">
                    {/* Nothing here */}
                  </div>
                </div>

                <div className="px-3 md:px-8">

                
                {data.length !== 0 ? (

        <Suspense fallback={<Skeleton />}>
                          <DataTable
                            data={data}
                            columns={columns}
                            title={t('List of coordinators')}
                          />

                          </Suspense>
                          ) : (

                            <div className="text-center mt-7 text-lg uppercase">
                              <p> {t('No coordinator found')}</p>
                            </div>
                          )}
                </div>


              </div>
            </div>
          </div>
        </div>
      </div>
    </>
  );
};

export default AdminTraineeDashboard;
