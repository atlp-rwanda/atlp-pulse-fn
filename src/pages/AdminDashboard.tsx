import React, { useContext, useEffect, useState } from 'react';
import { t } from 'i18next';
import { useTranslation } from 'react-i18next';
import { useMutation } from '@apollo/client';
import { toast } from 'react-toastify';
// eslint-disable-next-line import/no-useless-path-segments
import useDocumentTitle from '../hook/useDocumentTitle';
import Comingsoon from './Comingsoon';
import Button from '../components/Buttons';
import { UserContext } from '../hook/useAuth';
import { INVITE_USER_MUTATION } from '../Mutations/manageStudentMutations';

function SupAdDashboard() {
  const { user } = useContext(UserContext);
  const { t }: any = useTranslation();

  const organizationToken = localStorage.getItem('orgToken');

  const [inviteTraineeModel, setInviteTraineeModel] = useState(false);
  const [inviteEmail, setInviteEmail] = useState('');
  const [buttonLoading, setButtonLoading] = useState(false);

  const inviteModel = () => {
    const newState = !inviteTraineeModel;
    setInviteTraineeModel(newState);
    //  this is true
  };

  const [inviteUser] = useMutation(INVITE_USER_MUTATION, {
    variables: {
      email: inviteEmail,
      orgToken: organizationToken,
      type: 'organisation',
    },
    onCompleted: (data) => {
      setTimeout(() => {
        setButtonLoading(false);
        toast.success(data.inviteUser);
        inviteModel();
      }, 1000);
    },
    onError: (err) => {
      setTimeout(() => {
        setButtonLoading(false);
        toast.error(err.message);
      }, 1000);
    },
  });

  useDocumentTitle('Dashboard');

  useEffect(() => {
    localStorage.setItem('inviteEmail', inviteEmail);
  }, [inviteEmail]);
  return (
    <>
      {/* =========================== Start::  InviteTraineeModel =============================== */}

      <div
        className={`h-screen w-screen bg-black bg-opacity-30 backdrop-blur-sm fixed top-0 left-0 z-20 flex items-center justify-center  px-4 ${
          inviteTraineeModel === true ? 'block' : 'hidden'
        }`}
      >
        <div className="bg-indigo-100 dark:bg-dark-bg w-full sm:w-3/4  xl:w-4/12 rounded-lg p-4 pb-8">
          <div className="card-title w-full flex  flex-wrap justify-center items-center  ">
            <h3 className="font-bold text-sm dark:text-white text-center w-11/12 ">
              {t('Send Invitation')}
            </h3>
            <hr className=" bg-primary border-gray-400 my-3 w-full" />
          </div>
          <div className="card-body">
            <form className=" py-3 px-8">
              <div className="card-title w-full flex  flex-wrap justify-center items-center  ">
                <h3 className="font-bold text-sm dark:text-white text-center w-11/12 ">
                  {t(
                    'Fill in the email to invite an organisation to DevPulse.',
                  )}
                </h3>
              </div>

              <div className="text-white input my-3 h-9 ">
                <div className="text-white grouped-input flex items-center h-full w-full rounded-md">
                  <input
                    data-testid="inviteInput"
                    value={inviteEmail}
                    onChange={(e) => {
                      setInviteEmail(e.target.value);
                    }}
                    type="email"
                    name="email"
                    className=" dark:bg-dark-tertiary text-black border border-primary py-2 rounded outline-none px-5 font-sans text-xs w-full"
                    placeholder={t('email')}
                  />
                </div>
              </div>

              <div className="w-full flex justify-between">
                <Button
                  data-testid="removeInviteModel"
                  variant="info"
                  size="sm"
                  style="w-[30%] md:w-1/4 text-sm font-sans"
                  onClick={() => inviteModel()}
                >
                  {t('Cancel')}
                </Button>

                <Button
                  data-testid="invite"
                  variant="primary"
                  size="sm"
                  style="w-[30%] md:w-1/4 text-sm font-sans"
                  onClick={() => {
                    setButtonLoading(true);
                    inviteUser();
                  }}
                  loading={buttonLoading}
                >
                  {t('Invite')}
                </Button>
              </div>
            </form>
          </div>
        </div>
      </div>
      {/* =========================== End::  InviteTraineeModel =============================== */}

      <div className="flex flex-col grow bg-light-bg dark:bg-dark-frame-bg">
        <div className="flex flex-row pb-8 justify-center">
          <div className="w-[100%] h-[100%]">
            {/* <div className="grid grid-cols-2 mb-12 md:mb-24 lg:mb-0 lg:grid-cols-4">
            <Card text={t('Coordinators')} number={47} />
            <Card text={t('Trainees')} number={560} />
            <Card text={t('Cohorts')} number={8} />
            <Card text={t('Performance')} number={4} />
          </div>
          <Chart title={t('Overall performance')} /> */}
            <div>
              {/* <div className="bg-light-bg dark:bg-dark-frame-bg  ">
                <div className="flex items-left px-10 lg:px-60 pt-24 pb-8"> */}
              {/* <div className="flex gap-2 pt-5">
                {user?.role === 'coordinator' || undefined ? (
                  ''
                ) : (
                  <Button
                    variant="primary"
                    size="lg"
                    data-testid="inviteModel"
                    style="m-0"
                    onClick={inviteModel}
                  >
                    {t('Invite an organization')}
                  </Button>
                )}
              </div> */}
            </div>

            {/* </div>
              </div> */}

            <Comingsoon title="Dashboard" />
          </div>
        </div>
      </div>
    </>
  );
}

export default SupAdDashboard;