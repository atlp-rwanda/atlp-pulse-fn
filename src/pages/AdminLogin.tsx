import React from 'react';

const Login = () => {
    const handleFormSubmit = (e:any) => {
        e.preventDefault();

        let email = e.target.elements.email?.value;
        let password = e.target.elements.password?.value;

        console.log(email, password);
    };
    return (
        <div className='h-screen flex bg-light-bg dark:bg-[#262E3D]'>
            <div className='w-full max-w-2xl  m-auto bg-white dark:bg-[#1F2A37] rounded-lg py-12 px-14'>
                <h1 className='text-2xl font-medium text-primary mt-4 mb-12 text-center dark:text-white'>
                    Log in to your account
                </h1>

                <form onSubmit={handleFormSubmit}>
                    <div>
                        <label htmlFor='email' className='dark:text-white'>Email</label>
                        <input
                            type='email'
                            className={`w-full p-2 text-primary dark:text-white border rounded-md outline-none text-sm transition duration-150 ease-in-out mb-4 dark:border-white dark:bg-[#1F2A37]`}
                            id='email'
                            placeholder='Your Email'
                        />
                    </div>
                    <div>
                        <label htmlFor='password' className='dark:text-white'>Password</label>
                        <input
                            type='password'
                            className={`w-full p-2 text-primary dark:text-white border rounded-md outline-none text-sm transition duration-150 ease-in-out mb-4 dark:border-white dark:bg-[#1F2A37]`}
                            id='password'
                            placeholder='Your Password'
                        />
                    </div>

                    <div className='flex justify-center items-center mt-6'>
                        <button
                            className={`w-full py-2 px-4 text-sm text-white bg-[#148FB6] rounded border border-green focus:outline-none focus:border-green-dark`}
                        >
                            Login
                        </button>
                    </div>
                </form>
            </div>
        </div>
    );
};

export default Login;