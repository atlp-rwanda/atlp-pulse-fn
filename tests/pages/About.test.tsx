import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { MockedProvider as ApolloProvider } from '@apollo/client/testing';
import { About } from '../../src/pages/About';

describe('About page', () => {
  it('renders the about page', () => {
    const elem = renderer.create(
      <MemoryRouter>
        <ApolloProvider>
          <About />
        </ApolloProvider>
      </MemoryRouter>,
    );

    expect(elem.toJSON()).toMatchSnapshot();
  });
});
