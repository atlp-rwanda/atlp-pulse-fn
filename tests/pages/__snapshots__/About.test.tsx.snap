// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders the about page 1`] = `
<div
  className=" bg-white   mt-auto dark:bg-dark-frame-bg font-serif"
>
  <div
    className=" justify-between w-full h-full pb-5"
  >
    <div>
      <h1
        className="text-center text-neutral-700 mb-5 font lg:text-4xl md:text-2xl pt-10 pb-4  dark:text-slate-100"
      >
        hero2
      </h1>
    </div>
    <div
      className="mt-10 card-image lg:mx-10 sm:mx-5 md:flex flex-row "
    >
      <div
        className="shadow-md sm:shadow-none mx-auto md:mx-0 bg-indigo-100  items-center  md:items-start flex-col dark:bg-dark-bg md:w-[45%] md:py-10 md:px-4 max-w-[35em] sm:w-full   sm:rounded-3xl  md:mr-2"
      >
        <h2
          className="pt-3 header-style lg:px-5 lg:text-2xl sm:text-xl lg:text-start sm:text-center md:text-start dark:text-slate-200"
        >
          performance
        </h2>
        <div>
          <div
            className="xl:hidden md:hidden mb-0 mt-4"
          >
            <img
              alt="frame"
              src={Object {}}
            />
          </div>
          <p
            className="text-style mt-0 lg:px-5 bg-[#E0E7FF] dark:bg-dark-bg rounded-b-3xl md:rounded-none p-8 md:px-0 md:py-4 dark:text-slate-300"
          >
            body1
          </p>
        </div>
      </div>
      <div
        className="sm:hidden md:flex xl:flex lg:flex"
      >
        <img
          alt="frame"
          className="rounded-2xl dark:opacity-80 lg:max-h-[25em]"
          src={Object {}}
        />
      </div>
    </div>
    <div
      className="mt-10 card-image lg:mx-10 sm:mx-5 md:flex flex-row-reverse "
    >
      <div
        className="shadow-md sm:shadow-none mx-auto md:mx-0 bg-indigo-100  items-center  md:items-start flex-col dark:bg-dark-bg md:w-[45%] md:py-10 md:px-4 max-w-[35em] sm:w-full   sm:rounded-3xl  md:mr-2"
      >
        <h2
          className="pt-3 header-style lg:px-5 lg:text-2xl sm:text-xl lg:text-start sm:text-center md:text-start dark:text-slate-200"
        >
          analytics
        </h2>
        <div>
          <div
            className="xl:hidden md:hidden mb-0 mt-4"
          >
            <img
              alt="frame"
              src={Object {}}
            />
          </div>
          <p
            className="text-style mt-0 lg:px-5 bg-[#E0E7FF] dark:bg-dark-bg rounded-b-3xl md:rounded-none p-8 md:px-0 md:py-4 dark:text-slate-300"
          >
            body2
          </p>
        </div>
      </div>
      <div
        className="sm:hidden md:flex xl:flex lg:flex"
      >
        <img
          alt="frame"
          className="rounded-2xl dark:opacity-80 lg:max-h-[25em]"
          src={Object {}}
        />
      </div>
    </div>
    <div
      className="mt-10 card-image lg:mx-10 sm:mx-5 md:flex flex-row "
    >
      <div
        className="shadow-md sm:shadow-none mx-auto md:mx-0 bg-indigo-100  items-center  md:items-start flex-col dark:bg-dark-bg md:w-[45%] md:py-10 md:px-4 max-w-[35em] sm:w-full   sm:rounded-3xl  md:mr-2"
      >
        <h2
          className="pt-3 header-style lg:px-5 lg:text-2xl sm:text-xl lg:text-start sm:text-center md:text-start dark:text-slate-200"
        >
          continuos
        </h2>
        <div>
          <div
            className="xl:hidden md:hidden mb-0 mt-4"
          >
            <img
              alt="frame"
              src={Object {}}
            />
          </div>
          <p
            className="text-style mt-0 lg:px-5 bg-[#E0E7FF] dark:bg-dark-bg rounded-b-3xl md:rounded-none p-8 md:px-0 md:py-4 dark:text-slate-300"
          >
            body3
          </p>
        </div>
      </div>
      <div
        className="sm:hidden md:flex xl:flex lg:flex"
      >
        <img
          alt="frame"
          className="rounded-2xl dark:opacity-80 lg:max-h-[25em]"
          src={Object {}}
        />
      </div>
    </div>
    <div
      className="mt-10 card-image lg:mx-10 sm:mx-5 md:flex flex-row-reverse "
    >
      <div
        className="shadow-md sm:shadow-none mx-auto md:mx-0 bg-indigo-100  items-center  md:items-start flex-col dark:bg-dark-bg md:w-[45%] md:py-10 md:px-4 max-w-[35em] sm:w-full   sm:rounded-3xl  md:mr-2"
      >
        <h2
          className="pt-3 header-style lg:px-5 lg:text-2xl sm:text-xl lg:text-start sm:text-center md:text-start dark:text-slate-200"
        >
          goal
        </h2>
        <div>
          <div
            className="xl:hidden md:hidden mb-0 mt-4"
          >
            <img
              alt="frame"
              src={Object {}}
            />
          </div>
          <p
            className="text-style mt-0 lg:px-5 bg-[#E0E7FF] dark:bg-dark-bg rounded-b-3xl md:rounded-none p-8 md:px-0 md:py-4 dark:text-slate-300"
          >
            body4
          </p>
        </div>
      </div>
      <div
        className="sm:hidden md:flex xl:flex lg:flex"
      >
        <img
          alt="frame"
          className="rounded-2xl dark:opacity-80 lg:max-h-[25em]"
          src={Object {}}
        />
      </div>
    </div>
    <div
      className="px-5 mt-20"
    >
      <h1
        className="font text-center text-neutral-700 lg:text-4xl sm:text-2xl dark:text-slate-100"
      >
         
        Come shape the future together
      </h1>
    </div>
    <div
      className="block md:hidden flex-col lg:mx-10 md:mx-20 my-20 gap-10 relative font-serif"
    >
      <div
        className="bg-indigo-100 dark:bg-dark-bg  dark:text-slate-300 lg:w-1/3 p-8 md:w-full rounded-b-3xl sm:mx-3 rounded-t-3xl visible"
      >
        <div
          className="flex flex-col sm:flex-row  mb-6 items-center"
        >
          <img
            alt=""
            className="sm:w-1/4"
            src={Object {}}
          />
          <ul>
            <li
              className="text-xs ml-3  dark:text-slate-300 text-neutral-600"
            >
              Bernard Dushimimana
            </li>
            <li
              className="text-xs mt-2  dark:text-slate-300 ml-3"
            >
              Sr.Manager
              , 
              Andela
            </li>
          </ul>
        </div>
        <p
          className="text-base  dark:text-slate-300 text-neutral-900"
        >
          Content1
        </p>
      </div>
      <div
        className="bg-indigo-100 dark:bg-dark-bg  dark:text-slate-300 lg:w-1/3 p-8 md:w-full rounded-b-3xl sm:mx-3 rounded-t-3xl hidden"
      >
        <div
          className="flex flex-col sm:flex-row  mb-6 items-center"
        >
          <img
            alt=""
            className="sm:w-1/4"
            src={Object {}}
          />
          <ul>
            <li
              className="text-xs ml-3  dark:text-slate-300 text-neutral-600"
            >
              Susan
            </li>
            <li
              className="text-xs mt-2  dark:text-slate-300 ml-3"
            >
              Sr.Manager
              , 
              Andela
            </li>
          </ul>
        </div>
        <p
          className="text-base  dark:text-slate-300 text-neutral-900"
        >
          Content2
        </p>
      </div>
      <div
        className="bg-indigo-100 dark:bg-dark-bg  dark:text-slate-300 lg:w-1/3 p-8 md:w-full rounded-b-3xl sm:mx-3 rounded-t-3xl hidden"
      >
        <div
          className="flex flex-col sm:flex-row  mb-6 items-center"
        >
          <img
            alt=""
            className="sm:w-1/4"
            src={Object {}}
          />
          <ul>
            <li
              className="text-xs ml-3  dark:text-slate-300 text-neutral-600"
            >
              Dr. Jack
            </li>
            <li
              className="text-xs mt-2  dark:text-slate-300 ml-3"
            >
              Director
              , 
              University of Rwanda
            </li>
          </ul>
        </div>
        <p
          className="text-base  dark:text-slate-300 text-neutral-900"
        >
          Content3
        </p>
      </div>
      <span
        className="absolute top-1/2 left-3 transform -translate-y-1/2  text-neutral-600 rounded-full bg-slate-50 p-1 drop-shadow-2xl opacity-50 "
      >
        <svg
          className="cursor-pointer"
          fill="currentColor"
          height={20}
          onClick={[Function]}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 0 0 0 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
          />
        </svg>
      </span>
      <span
        className="absolute top-1/2 right-3 transform -translate-y-1/2 text-neutral-600 rounded-full bg-slate-50 p-1 drop-shadow-2xl opacity-50 "
      >
        <svg
          className="cursor-pointer"
          fill="currentColor"
          height={20}
          onClick={[Function]}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M765.7 486.8L314.9 134.7A7.97 7.97 0 0 0 302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 0 0 0-50.4z"
          />
        </svg>
      </span>
    </div>
    <div
      className="hidden md:flex md:flex-row flex-col  lg:mx-0 md:mx-0 my-20 gap-10 md:flex-wrap lg:flex-nowrap font-serif"
    >
      <div
        className="bg-indigo-100  dark:bg-dark-bg lg:w-1/3 p-8 md:w-full  rounded-b-3xl sm:mx-3 rounded-t-3xl "
      >
        <div
          className="flex flex-col sm:flex-row  mb-6 items-center"
        >
          <img
            alt=""
            className="md:w-1/5 sm:w-1/3"
            src={Object {}}
          />
          <ul>
            <li
              className="text-sm ml-3  dark:text-slate-300  text-neutral-600"
            >
              Bernard Dushimimana
            </li>
            <li
              className="text-sm  dark:text-slate-300  mt-2 ml-3"
            >
              Sr.Manager
              , 
              Andela
            </li>
          </ul>
        </div>
        <p
          className="text-base dark:text-slate-300  text-neutral-900"
        >
          Content1
        </p>
      </div>
      <div
        className="bg-indigo-100  dark:bg-dark-bg lg:w-1/3 p-8 md:w-full  rounded-b-3xl sm:mx-3 rounded-t-3xl "
      >
        <div
          className="flex flex-col sm:flex-row  mb-6 items-center"
        >
          <img
            alt=""
            className="md:w-1/5 sm:w-1/3"
            src={Object {}}
          />
          <ul>
            <li
              className="text-sm ml-3  dark:text-slate-300  text-neutral-600"
            >
              Susan
            </li>
            <li
              className="text-sm  dark:text-slate-300  mt-2 ml-3"
            >
              Sr.Manager
              , 
              Andela
            </li>
          </ul>
        </div>
        <p
          className="text-base dark:text-slate-300  text-neutral-900"
        >
          Content2
        </p>
      </div>
      <div
        className="bg-indigo-100  dark:bg-dark-bg lg:w-1/3 p-8 md:w-full  rounded-b-3xl sm:mx-3 rounded-t-3xl "
      >
        <div
          className="flex flex-col sm:flex-row  mb-6 items-center"
        >
          <img
            alt=""
            className="md:w-1/5 sm:w-1/3"
            src={Object {}}
          />
          <ul>
            <li
              className="text-sm ml-3  dark:text-slate-300  text-neutral-600"
            >
              Dr. Jack
            </li>
            <li
              className="text-sm  dark:text-slate-300  mt-2 ml-3"
            >
              Director
              , 
              University of Rwanda
            </li>
          </ul>
        </div>
        <p
          className="text-base dark:text-slate-300  text-neutral-900"
        >
          Content3
        </p>
      </div>
    </div>
  </div>
</div>
`;
